Alpine Node-Red

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

alpine-node-red

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/alpine-node-red.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Alpine

### 📝 Provide a clear and concise description of the issue.

script fails and no autologin / login incorrect

### 🔄 Steps to reproduce the issue.

just run script

### ❌ Paste the full error output (if available).

  ✔️  Created Node-RED User

added 278 packages in 21s

50 packages are looking for funding
  run `npm fund` for details
  ✔️  Installed Node-RED
   Creating Node-RED Service * service nodered added to runlevel default
  ✔️  Created Node-RED Service
   Starting Node-RED * Caching service dependencies ...                          [ ok ]
 * Starting nodered ...
 * start-stop-daemon: /usr/bin/node-red does not exist
 * Failed to start nodered                                                       [ !! ]
 * ERROR: nodered failed to start

[ERROR] in line 52: exit code 1: while executing command $STD service nodered start

/dev/fd/63: line 73: SPINNER_PID: unbound variable

### 🖼️ Additional context (optional).

_No response_

--- a/install/alpine-node-red-install.sh
+++ b/install/alpine-node-red-install.sh
@@ -35,7 +35,7 @@ service_path="/etc/init.d/nodered"
 echo '#!/sbin/openrc-run
 description="Node-RED Service"
 
-command="/usr/bin/node-red"
+command="/usr/local/bin/node-red"
 command_args="--max-old-space-size=128 -v"
 command_user="nodered"
 pidfile="/var/run/nodered.pid"

--- a/install/alpine-node-red-install.sh
+++ b/install/alpine-node-red-install.sh
@@ -29,6 +29,12 @@ msg_info "Installing Node-RED"
 npm install -g --unsafe-perm node-red
 msg_ok "Installed Node-RED"
 
+msg_info "Creating /home/nodered"
+mkdir -p /home/nodered
+chown -R nodered:users /home/nodered
+chmod 750 /home/nodered
+msg_ok "Created /home/nodered"
+
 msg_info "Creating Node-RED Service"
 service_path="/etc/init.d/nodered"
 

